### DISCUSSION TITLE
running on mini touch screen

### DISCUSSION THREAD

**Discussion by mak333333** [HemantKArya/BloomeeTunes#78] [Show and tell]
i love this app i use it on a 3inch by 11inch touch display and the works and fits the best from all other apps I've tried. using borderless gaming app to make it Fullscreen and cant use the lyrics feature or the in app volume control but that's ok still the best app ive found for this use case
 ![PXL_20241211_184152672 MP](https://github.com/user-attachments/assets/94629afe-aca1-48c1-9f19-2e6bc5bd1837)
![PXL_20241211_184206936 MP](https://github.com/user-attachments/assets/72612d71-532b-4258-9590-a5ecb34f5c48)

**Comment by HemantKArya** [comment]
THanx @mak333333 😊. yeah sure I guess we have to work on the design more to support more wider screens to support cover arts and lyrics support.
